### PR TITLE
[Draft] Cleaning up python script in #885

### PR DESCRIPTION
**Pull request by jbzdarkid**

### CODE REVIEW

File: scripts/update_xcode_config_cpp_checks.py [hunk 1]
```diff
@@ -1,27 +1,61 @@
 import os
```
Comment (kaisong1990) [inline scripts/update_xcode_config_cpp_checks.py:1]: This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.

**Comment by jbzdarkid** [comment]
@kaisong1990 Looks like the new script is working. If you like the changes, please merge this into your branch:
```
git fetch
git checkout kaisong1990/update_xcode_conf_cpp
git merge --squash jbzdarkid/test
```

**Comment by kaisong1990** [comment]
> @kaisong1990 Looks like the new script is working. If you like the changes, please merge this into your branch:
> 
> ```
> git fetch
> git checkout kaisong1990/update_xcode_conf_cpp
> git merge --squash jbzdarkid/test
> ```

Looks good to me, will do it

**Comment by kaisong1990** [comment]
@jbzdarkid Feel free to close this one as your changes had been merged in

**Pull request closed without merging**